src/transformers/pipelines/text_to_speech.py

```diff
@@ -0,0 +1,168 @@
+from typing import List, Union
+
+import torch
+from datasets import load_dataset
+
+from transformers import Pipeline, SpeechT5HifiGan
+
+
+ONLY_ONE_SPEAKER_EMBEDDINGS_LIST = ["bark"]
+
+SPEAKER_EMBEDDINGS_KEY_MAPPING = {"bark": "history_prompt"}
```

Comment:
`Bark` speaker embeddings attribute name was not consistent throughout the input->Pipeline->Model pass.